src/lib/components/workflow/advanced-filter/status-filter.svelte

```diff
@@ -3,19 +3,20 @@
   import Option from '$lib/holocene/select/option.svelte';
   import WorkflowStatus from '$lib/components/workflow-status.svelte';
   import KeywordConditionals from './keyword-conditionals.svelte';
+  import { translate } from '$lib/i18n/translate';
 
   export let value = '';
   export let conditional = '';
 
   const statuses = {
-    All: 'All',
-    Running: 'Running',
-    'Timed Out': 'TimedOut',
-    Completed: 'Completed',
-    Failed: 'Failed',
-    'Continued as New': 'ContinuedAsNew',
-    Canceled: 'Canceled',
-    Terminated: 'Terminated',
+    All: translate('all'),
```

Comment:
Do we need `translate` here? We are already translating `humanFriendlyNames` in `src/lib/components/workflow-status.svelte`.

Reply:
Good catch, I didn't realize that neither the keys or values of this object are actually displayed in the DOM